File: AndroidBasics/notes/src/main/java/pan/alexander/notes/presentation/fragments/SettingsFragment.java
package pan.alexander.notes.presentation.fragments;

import android.os.Bundle;

import androidx.preference.PreferenceFragmentCompat;

import pan.alexander.notes.R;

public class SettingsFragment extends PreferenceFragmentCompat {

    @Override
    public void onCreatePreferences(Bundle savedInstanceState, String rootKey) {
        setPreferencesFromResource(R.xml.root_preferences, rootKey);
    }
}

File: Filmrevealer/app/src/main/java/pan/alexander/filmrevealer/presentation/fragments/SettingsFragment.kt
package pan.alexander.filmrevealer.presentation.fragments

import android.os.Bundle
import androidx.preference.PreferenceFragmentCompat
import pan.alexander.filmrevealer.R

class SettingsFragment : PreferenceFragmentCompat() {
    override fun onCreatePreferences(savedInstanceState: Bundle?, rootKey: String?) {
        setPreferencesFromResource(R.xml.root_preferences, rootKey)
    }
}
